src/Psy/Input/WholeStringInput.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Psy\Input;
+
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\StringInput;
+
+/**
+ * Class WholeStringInput
+ * @package Psy\Input
+ */
+class WholeStringInput extends StringInput
```

Comment:
:+1: This seems like a good approach to the issue.

It could also be extended in the future if we have a whole string input command that uses flags, so we could do something like `timeit --verbose foo()` and be able to process options as well. But not yet :)